pipeline/scripts/AddOnFunctions/output_Helix.R

```diff
@@ -0,0 +1,35 @@
+output_for_Helix <- function(protocol_name, df_metabs_Helix){
+    
+  # Remove positive controls
+  df_metabs_Helix <- df_metabs_Helix %>% filter(grepl("^P[0-9]{4}M",Patient))
+  
+  # Add 'Vial' column, each patient has unique ID
+  df_metabs_Helix <- df_metabs_Helix %>% 
+    group_by(Patient) %>% 
+    mutate(Vial = cur_group_id()) %>% 
+    ungroup()
+  
+  # Split patient code into labnummer and Onderzoeksnummer
+  for (row in 1:nrow(df_metabs_Helix)) {
+    df_metabs_Helix[row,"labnummer"] <- gsub("^P|\\.[0-9]*", "", df_metabs_Helix[row,"Patient"])
+    labnummer_split <- strsplit(as.character(df_metabs_Helix[row, "labnummer"]), "M")[[1]]
+    df_metabs_Helix[row, "Onderzoeksnummer"] <- paste0("MB",labnummer_split[1],"/",labnummer_split[2])
```

Comment:
Would be nice to create a function `get_lab_id_and_...`.  This would prepare the code for unit testing.

Reply:
Agree, added function